inst/create-list-available-forecasts.R

```diff
@@ -24,6 +24,9 @@ usethis::use_data(metrics_sample, overwrite = TRUE)
 
 metrics_quantile <- list(
   "wis" = wis,
+  "overprediction" = overprediction,
+  "underprediction" = underprediction,
+  "dispersion" = dispersion,
```

Comment:
I'm seeing from this that this PR depends on a totally random branch and not dev. Why?